### DISCUSSION TITLE
Let "completeCartWorkflow" produce "orderCreated" hook

### DISCUSSION THREAD

**Discussion by IgorKhomenko** [medusajs/medusa#9765] [Feature Requests]
Currently, an order can be created in 2 ways:

- via `createOrdersWorkflow`
- via `completeCartWorkflow`

The issue with `completeCartWorkflow` is that it does not produce any hook with created order ID, so a user will need to come up with some workarounds

**Comment by JakeCoxon** [comment]
How do you workaround this?

**Reply by IgorKhomenko** [reply]
For now I use a subscriber on order.placed event.

Not ideal since it's lack of user context, but better than nothing. 

At some point I'm gonna create a patch to medusa code to bring the hook

**Reply by JakeCoxon** [reply]
I appreciate the quick reply! `order.placed` will work for me, thanks

**Comment by Avia-Code** [comment]
Hey @IgorKhomenko , I agree that this workflow was missing a hook, so I added it https://github.com/medusajs/medusa/pull/11660.

**Reply by IgorKhomenko** [reply]
great , thanks for the fast contribution !
